packages/oauth/oauth-provider/src/metadata/build-metadata.ts

```diff
@@ -62,15 +35,15 @@ export function buildMetadata(
     response_types_supported: [
       // OAuth
       'code',
-      'token',
+      // 'token',
 
       // OpenID
-      'none',
-      'code id_token token',
-      'code id_token',
-      'code token',
-      'id_token token',
-      'id_token',
+      // 'none',
+      // 'code id_token token',
+      // 'code id_token',
+      // 'code token',
+      // 'id_token token',
+      // 'id_token',
```

Comment:
Can we remove these entirely, or prefer to keep them around commented for some purpose?

Reply:
I kept them around as a way to state that we know about those other values but we explicitly do not support them.